<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Explorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutLeft">
          <item>
           <widget class="QComboBox" name="cbDriveLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leDriveInfoLeft">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbRootLeft">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>\</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbUpLeft">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>..</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lePathLeft">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeViewLeft">
          <property name="indentation">
           <number>20</number>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutRight">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutRight">
          <item>
           <widget class="QComboBox" name="cbDriveRight">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leDriveInfoRight">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbRootRight">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>\</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbUpRight">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>..</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lePathRight">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeViewRight">
          <property name="indentation">
           <number>20</number>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Files">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Commands">
    <property name="title">
     <string>&amp;Commands</string>
    </property>
   </widget>
   <addaction name="menu_Files"/>
   <addaction name="menu_Commands"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbDriveLeft</tabstop>
  <tabstop>cbDriveRight</tabstop>
  <tabstop>treeViewLeft</tabstop>
  <tabstop>treeViewRight</tabstop>
  <tabstop>lePathLeft</tabstop>
  <tabstop>pbRootLeft</tabstop>
  <tabstop>leDriveInfoLeft</tabstop>
  <tabstop>leDriveInfoRight</tabstop>
  <tabstop>pbRootRight</tabstop>
  <tabstop>pbUpRight</tabstop>
  <tabstop>lePathRight</tabstop>
  <tabstop>pbUpLeft</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
